app/routers/books.py

```diff
@@ -20,32 +25,40 @@
 # Local application imports
 # -------------------------
 from config import settings
-
 from ..applogger import rslogger
 from ..crud import create_useinfo_entry
 from ..db import database as db
 from ..schemas import LogItem, LogItemIncoming
 
-#
+# Routing
+# =======
 # Setup the router object for the endpoints defined in this file.  These will
-# be connected to the main application in main.py
-#
+# be `connected <included routing>` to the main application in `../main.py`.
 router = APIRouter(
-    prefix="/books",  # shortcut so we don't have to repeat this part
-    tags=["books"],  # groups all logger tags together in the docs
+    # shortcut so we don't have to repeat this part
+    prefix="/books",
+    # groups all logger `tags <https://fastapi.tiangolo.com/tutorial/path-operation-configuration/#tags>`_ together in the docs.
+    tags=["books"],
 )
 
 
-# Static Asset renderers
-# https://fastapi.tiangolo.com/tutorial/static-files/?h=+staticfiles#use-staticfiles
-# https://fastapi.tiangolo.com/advanced/custom-response/?h=+filere#fileresponse
-# for paths like: /books/published/basecourse/_static/rest
+# Options for static asset renderers:
+#
+# - `StaticFiles <https://fastapi.tiangolo.com/tutorial/static-files/?h=+staticfiles#use-staticfiles>`_. However, this assumes the static routes are known *a priori*, in contrast to books (with their static assets) that are dynamically added and removed.
+# - Manually route static files, returning them using a `FileResponse <https://fastapi.tiangolo.com/advanced/custom-response/#fileresponse>`_. This is the approach taken.
+#
+# for paths like: ``/books/published/basecourse/_static/rest``.
 # If it is fast and efficient to handle it here it would be great.  We currently avoid
-# any static file contact with web2py and handle static files upstream with nginx directly
-# Note the use of the path type for filepath in the decoration.  If you don't use path it
-# seems to only get you the ``next`` part of the path /pre/vious/next/the/rest
+# any static file contact with web2py and handle static files upstream with nginx directly; therefore, this is useful only for testing/a non-production environment.
+# Note the use of the ``path``` type for filepath in the decoration.  If you don't use path it
+# seems to only get you the ``next`` part of the path ``/pre/vious/next/the/rest``.
+#
+# :index:`todo`: **Routes for draft (instructor-only) books.**
 @router.get("/published/{course:str}/_static/{filepath:path}")
 async def get_static(course: str, filepath: str):
+    # :index:`fixme`: **Find a way to avoid a hard-coded path.** This fix is needed for the route below as well.
```

Comment:
This is still a prototype.... I think should make a configuration variable like BOOK_LIBRARY  that points to a folder that holds all of the books.  The great thing is it does not have to be inside the RunestoneServer folder!

Reply:
That makes sense. I'll leave the comment in so we can find this when it's time to fix. Just more code review comments...